apps/spline.c

```diff
@@ -15,11 +15,11 @@
 
 #define _apps_spline_pixmap(spline) ((spline)->widget.window->pixmap)
 
-#define NPT 4
 
 typedef struct _apps_spline {
     twin_widget_t widget;
-    twin_point_t points[NPT];
+    int npt;
```

Comment:
The variable `npt` is defined as `npoints` in `./src/path.c`. To maintain consistency, it is recommended to rename it to `npoints` or `n_points`. Using the latter can make it more distinct from points.